dockerfiles/Dockerfile.training

```diff
@@ -21,11 +21,14 @@ WORKDIR /stage
 
 # install curl and git
 RUN apt-get -y update &&\
+    apt-get install -y sudo git bash unattended-upgrades && \
     apt-get -y --no-install-recommends install \
         curl \
         git \
         language-pack-en
 
+RUN unattended-upgrade
```

Comment:
Can you add a comment line above explaining the purpose of the command? e.g. "# force update existing packages to resolve security vulnerabilities."

Reply:
It's a good idea. Added